env0/utils.go

```diff
@@ -453,3 +457,34 @@ func writeResourceDataEx(prefix string, i interface{}, d *schema.ResourceData) e
 	}
 	return writeResourceDataSlice([]interface{}{i}, prefix, d)
 }
+
+func ttlToDuration(ttl string) (time.Duration, error) {
+	if ttl == "Infinite" || ttl == "inherit" {
+		return math.MaxInt64, nil
+	}
+
+	match := matchTtl.FindStringSubmatch(ttl)
+	if match == nil {
+		return 0, fmt.Errorf("invalid TTL format %s", ttl)
+	}
+
+	numberStr := match[1]
+	number, err := strconv.Atoi(numberStr)
+	if err != nil {
+		return 0, fmt.Errorf("invalid TTL format %s - not a number: %w", ttl, err)
+	}
+	// M/w/d/h
+
+	var hours int = number
+
+	switch rangeType := match[2]; rangeType {
+	case "M":
```

Comment:
Assumed that 1 Month equals 30 days.

Reply:
Isn't there a way to calc a "real" month using https://pkg.go.dev/time?
This will create problems

Reply:
1-M would mean something different every month.
In other words, 1-M isn't well-defined when it's configured.

Reply:
nvm, just keep it